WordPress/Classes/ViewRelated/Blog/Blog Details/Detail Header/BlogDetailHeaderView.swift

```diff
@@ -92,6 +92,10 @@ class BlogDetailHeaderView: UIView, BlogDetailHeader {
         siteIconView.spotlightIsShown = QuickStartTourGuide.shared.isCurrentElement(.siteIcon)
     }
 
+    @objc func updateLoadingTitle(isLoading: Bool) {
```

Comment:
How about `setTitleLoading(_ isLoading: Bool)`? (Would prevent the duplication of 'loading' when calling it, and I think `setTitleLoading` may be a little clearer than the existing about what it's doing?)

Reply:
Done in 1e49f70d7941a7b87ab0c07795ec71643224fcac